src/Http/GraphQLController.php

```diff
@@ -28,18 +31,35 @@ public function __construct(GraphQLService $graphqlService)
 
     /**
      * @param Request $request
-     * @return JsonResponse
+     *
+     * @return SymfonyJsonResponse
      */
-    public function process(Request $request)
+    public function handle(Request $request)
     {
         $processor = $this->graphqlService->getProcessor();
 
         $query     = $request->get('query');
-        $variables = $request->has('variables') ? $request->get('variables') : [];
+        $variables = $request->get('variables', []);
 
         $responseData = $processor->processPayload($query, $variables)->getResponseData();
 
-        return response()->json($this->responseDataToJson($responseData));
+        if (isset($responseData['exceptions'])) {
+            $request->attributes->set($this->getErrorAttribute(), new GraphQLError(
```

Comment:
@crisu83 @Jalle19 I just made the error attribute configurable here.

Reply:
+1